test/integration/test_cli_commands.py

```diff
@@ -874,3 +874,20 @@ def test_logging_with_logformat_logstash_flag(self):
                     obj={"filters":[]})
         output = sorted(result.output.splitlines(), reverse=True)[:4]
         self.assertEqual(expected, output)
+
+class TestCLIRepositoryCreate(CuratorTestCase):
+    def test_create_fs_repository(self):
+        test = clicktest.CliRunner()
+        result = test.invoke(
+                    curator.repomgrcli,
+                    [
+                        '--logfile', os.devnull,
+                        '--host', host,
```

Comment:
`host` and `port` are defined by environment variables, so that's how that's taken care of.  Again, it's manually specified, and that's what we're looking for here.  It overrides the default, even if it overrides it with the same value.